### PR TITLE
Turn ContactImporter into ES6 class

### LINKED ISSUE #532: ES6-ify contact importer
#### Issue Summary

Refactor the contact importer to make more use of ES6 standards

### PR DESCRIPTION
**Pull request by sethetter**
Also add `sendgrid` as a dependency so the test will run and pass.

Resolves #532

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=542) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-nodejs?pullRequest=542) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-nodejs?pullRequest=542) it.</sub>

File: packages/contact-importer/src/importer.spec.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+const sendgrid = require('sendgrid');
```
Comment (rafaelverger) [inline packages/contact-importer/src/importer.spec.js]: Why?

Comment (sethetter) [reply packages/contact-importer/src/importer.spec.js]: The test file uses sendgrid, but it wasn't in the file. This was required to get it to run and pass.

Comment (rafaelverger) [reply packages/contact-importer/src/importer.spec.js]: I would also add this script shortcut in main package json:
```
{
  // ...
  scripts: {
    // ... 
    "test:contact": "babel-node ./node_modules/istanbul/lib/cli cover ./node_modules/mocha/bin/_mocha \"packages/contact-importer/**/*.spec.js\"",
  }
}
```

Comment (sethetter) [reply packages/contact-importer/src/importer.spec.js]: Good call, I'll get that added shortly.

**Comment by sethetter** [comment]
@rafaelverger ready for another review!

**Comment by rafaelverger** [comment]
All of them make sense to me... @mbernier ?

**Comment by mbernier** [comment]
This is awesome!! Thank you @sethetter and @rafaelverger -- SO AWESOME seeing you guys take this one on. Thank you Thank you Thank you!!

**Comment by thinkingserious** [comment]
Hello @sethetter,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by sethetter** [comment]
@mbernier @thinkingserious thank you both! Happy to contribute. Thanks @rafaelverger for the review!

**Pull request merged**